charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -23,7 +23,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 36.6.2
+version: 36.6.3
```

Comment:
as minor versions of subcharts are raised i would raise minor version of kps too...

Reply:
@monotek 
Bumped minor version